Dockerfile

```diff
@@ -276,6 +276,10 @@ RUN pip install --upgrade cython && \
     pip install google-cloud-language==2.* && \
     pip install google-cloud-videointelligence==2.* && \
     pip install google-cloud-vision==2.* && \
+    # After launch install main
+    pip install git+https://github.com/googleapis/python-aiplatform.git@mb-release && \ 
+    pip install --upgrade pandas && \
```

Comment:
This is probably breaking other packages. `pandas` is used by many packages, if you need a newer version, let me know and I will look into upgrading the DLVM base image to get a newer version.

I also see a comment below about removing this.

Reply:
I tried a couple different configurations and I *think* this one can be removed, I'll give it a shot